BraveRewardsUI/Settings/Ads/Details/AdsDetailsViewController.swift

```diff
@@ -82,14 +96,72 @@ class AdsDetailsViewController: UIViewController {
     Strings.fourAdsPerHour,
     Strings.fiveAdsPerHour
   ]
-  
+
+  private let subdivisionTargetingOptions: KeyValuePairs = [
```

Comment:
Why use `KeyValuePairs` here? `KeyValuePairs` is less efficient/performant and since you are using hashable keys there's no reason to use it

Reply:
Discussed and staying with `KeyValuePairs`